<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Geetha" failures="1" tests="3" name="Default test" time="20.102" errors="0" timestamp="2023-06-15T17:09:10 PDT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.936"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="9.546"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.391">
    <failure type="java.lang.AssertionError" message="expected [] but found [Login | Salesforce]">
      <![CDATA[java.lang.AssertionError: expected [] but found [Login | Salesforce]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.auto.alldefinitionFiles.LoginScripts.user_should_logged_out(LoginScripts.java:117)
at âœ½.user should logged out(file:///C:/Users/geeth/eclipse-workspace/CucumberFramework/src/test/resources/features/loginFeatures.feature:26)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
